website/src/components/SequenceDetailsPage/SequenceDataUI.tsx

```diff
@@ -44,12 +47,8 @@ export const SequenceDataUI: React.FC<Props> = ({
     dataUseTermsHistory.sort((a, b) => (a.changeDate > b.changeDate ? -1 : 1));
     const currentDataUseTerms = dataUseTermsHistory[0].dataUseTerms;
 
-    const dataUseTermsIndex = tableData.findIndex((entry) => entry.name === 'dataUseTerms');
-    if (dataUseTermsIndex !== -1) {
-        tableData[dataUseTermsIndex].value = currentDataUseTerms.type;
-    }
-
-    const isRestricted = currentDataUseTerms.type === 'RESTRICTED';
+    const dataUseTerms = tableData.find((entry) => entry.name === DATA_USE_TERMS_FIELD);
```

Comment:
```suggestion
    const dataUseTerms = tableData.find((entry) => entry.name === DATA_USE_TERMS_FIELD);
    // Calculation below is based on the assumption that dataUseTerms is a string
    // Hence assert to get loud error instead of quiet false data
    if (typeof dataUseTerms?.value !== 'string') {
        throw new Error('Code assumes dataUseTerms are of type string. If it isn't, then need to change the calculation of isRestricted.');
    }
```

Reply:
Sorry - already merged. If you still think it's necessary, you can open a separate PR.